vspace/src/main/java/edu/asu/diging/vspace/core/util/JsonHelper.java

```diff
@@ -0,0 +1,25 @@
+package edu.asu.diging.vspace.core.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonHelper {
+
+    private static ObjectMapper mapper;
+    
+    static
+    {
+        mapper = new ObjectMapper();
+    }
+    
+    public static String convertToJson(Object object) throws JsonProcessingException
+    {
+        String jsonResult="";
+        jsonResult=mapper.writeValueAsString(object);
+        return jsonResult;
+    }
+    
+    public static ObjectMapper getMapper() {
+        return mapper;
+    }
+}
```

Comment:
none of these should be static methods or variables; create a component or service.